packages/ecc-client-ga4gh-trs/src/components/trs-list/definition/trs-list.template.ts

```diff
@@ -0,0 +1,442 @@
+          <div>
+            <span>Author</span>
+            <custom-tooltip>
+              <svg
+                slot="label"
+                class="tooltipIcon"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 16 16"
+                width="16"
+                height="16"
+              >
+                <path
+                  d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8Zm8-6.5a6.5 6.5 0 1 0 0 13 6.5 6.5 0 0 0 0-13ZM6.5 7.75A.75.75 0 0 1 7.25 7h1a.75.75 0 0 1 .75.75v2.75h.25a.75.75 0 0 1 0 1.5h-2a.75.75 0 0 1 0-1.5h.25v-2h-.25a.75.75 0 0 1-.75-.75ZM8 6a1 1 0 1 1 0-2 1 1 0 0 1 0 2Z"
+                ></path>
+              </svg>
+              The author of the tool.
+            </custom-tooltip>
+          </div>
+          <fast-text-field
+            type="text"
+            name="author"
+            value="${(x) => x.filterParams.author}"
+            @input="${(x, c) => x.handleFilterParamChange(c.event)}"
+          ></fast-text-field>
+        </label>
+      </div>
+      <fast-button
+        class="filterContainer--button"
+        @click="${(x) => x.handleApplyFilter()}"
+        >Apply</fast-button
+      >
+      <fast-button
+        class="filterContainer--clear"
+        @click="${(x) => x.handleClearFilter()}"
+        >Clear</fast-button
+      >
+    </div>
+  `
+)}`;
+
+const accordionTemplate = html<TRS>`
+  <fast-accordion expand-mode="multi">
+    ${repeat(
+      (x) => x.tools,
+      html<ITool>` <fast-accordion-item class="accordionItem">
+        <table class="styled-table">
+          <thead>
+            <tr>
+              <th>ID</th>
+              <th>Name</th>
+              <th>Meta Version</th>
+              <th>Tool Class</th>
+              <th>Description</th>
+              <th>Organization</th>
+              <th>URL</th>
+              <th>Checker</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr class="active-row">
+              <td>${(x) => x.id}</td>
+              <td>${(x) => x.name}</td>
+              <td>${(x) => x.meta_version}</td>
+              <td>${(x) => x.toolclass.name}</td>
+              <td>${(x) => x.description}</td>
+              <td>${(x) => x.organization}</td>
+              <td><a href=${(x) => x.url}>${(x) => x.url}</a></td>
+              <td>${(x) => (x.has_checker ? x.checker_url : "No checker")}</td>
+            </tr>
+          </tbody>
+        </table>
+        <div>
+          ${when(
+            (x) => x && x.aliases && x.aliases.length !== 0,
+            html` <h3>Aliases:</h3> `
+          )}
+          ${repeat(
+            (x) => x.aliases,
+            html<string>` <custom-copy
+                value="${(alias) => alias}"
+              ></custom-copy>
+              <div class="space"></div>`
+          )}
+        </div>
+        <div class="space"></div>
+        <h3>Versions:</h3>
+        <fast-tabs orientation="vertical">
+          ${repeat(
+            (x) => x.versions,
```

Comment:
for the hover effect on the versions don't use the color white, it blends in with the background, maybe use black.